operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_CPS.md

```diff
@@ -12,16 +12,16 @@ This page covers the appropriate contents for a BOS session template in order to
   - [`<transport>`](#transport)
   - [`<api_gateway>`](#api_gateway)
   - [`<timeout>`](#timeout)
-  - [`<etag>`](#etag)
   - [`<interface>[,<interface>][,<interface>]...`](#interfaceinterfaceinterface)
   - [`<ramroot>`](#ramroot)
   - [Example `rootfs_provider_passthrough`](#example-rootfs_provider_passthrough)
-- [`root=` kernel parameter](#root-kernel-parameter)
+- [Example kernel parameter](#example-kernel-parameter)
```

Comment:
I'm not sure why this link is here. As far as I can tell, the heading "Example kernel parameter" is nested underneath the "Appendix: `root=` kernel parameter.

Reply:
Removed.